mystery-deckbuilder/Assets/Scripts/Notepad/Notepad.cs

```diff
@@ -0,0 +1,270 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+
+public class Notepad : MonoBehaviour
+{
+    //Variables
+    public float numpages;
+    public float numchapters;
+
+    public GameObject notepadCanvas;
+    public GameObject pageImage;
+
+    public bool notebookActive;
+    public int currentChapterID = 0;
```

Comment:
This value should be set as a GameStateValue, so other scripts can access it easily and be notified if it changes. Check GameState.cs and GameStateValue.cs. If you need some help on this, let me know